### DISCUSSION TITLE
Why is `Field` missing the `example` keyword?  How to make `mypy` happy?

### DISCUSSION THREAD

**Discussion by JakeSummers** [pydantic/pydantic#10881] [Question]
I have a couple of dataclasses like this: 

```py
class ingredient(BaseModel):
    name: str

class FoodStuff(BaseModel):
    in_fridge: list[ingredient] = Field(
        ...,
        example=[ingredient(name="ham"), ingredient(name="milk")],
    )
    shopping_list: list[ingredient] = Field(
        ...,
        example=[ingredient(name="cheese")],
    )
```

I am using Pydantic with FastApi and need to set `example` to populate the swagger-ui: 

![image](https://github.com/user-attachments/assets/b8edfc3f-9e30-4cd2-84fa-0aaa234a1a04)

When I run `mypy` on this code, I get this error: 

```
 error: Unexpected keyword argument "example" for "Field"; did you mean "examples"?  [call-arg]
```

I know that the `examples` keyword exists, but that seems to be currently incompatible with FastApi.  

What is going on here?

**Comment by Viicos** [comment]
This is a documented feature of FastAPI: https://fastapi.tiangolo.com/tutorial/schema-extra-example/?h=examp#field-additional-arguments.

Make sure you are on the latest versions of FastAPI and Pydantic. You can also open an issue on the FastAPI issue tracker if you are confident something is wrong.
